ldm/dream/server.py

```diff
@@ -17,8 +17,8 @@ def do_GET(self):
                 self.wfile.write(content.read())
         else:
             path = "." + self.path
-            cwd = os.getcwd()
-            is_in_cwd = os.path.commonprefix((os.path.realpath(path), cwd)) == cwd
+            cwd = os.getcwd().lower()
+            is_in_cwd = os.path.commonprefix((os.path.realpath(path).lower(), cwd)) == cwd
```

Comment:
@david-ford Can you check if the following works instead?

```suggestion
            cwd = os.path.realpath(os.getcwd())
            is_in_cwd = os.path.commonprefix((os.path.realpath(path), cwd)) == cwd
```

If that works, it would be better, since then the check is still correct on case-sensitive filesystems.

Reply:
Yeah, this looks better because on Linux we could have ./static and ./Static side-by-side and they shouldn't both pass the test. Is there a specific test case for case-sensitive filesystems that I can check?

Reply:
I couldn't repro the issue personally, but a user tested my suggestion in https://github.com/lstein/stable-diffusion/issues/196#issuecomment-1230846125 and reports that it worked, so it should be ok. Do you want to just add that in while merging this PR? Alternatively I could open a new one with just that change.

Reply:
> Yeah, this looks better because on Linux we could have ./static and ./Static side-by-side and they shouldn't both pass the test. Is there a specific test case for case-sensitive filesystems that I can check?

I'm using Windows 10, with my environment setup following the instructions in the readme using Anaconda. When I do os.getcwd() I get c:\path\to\project and when I do os.realpath I get C:\path\to\project. 

I believe the suggestion here will work, but my PC is currently training. Strangely, the original dataset has no idea what a scythe is.

As mentioned below, I'll pull the unrelated changes from this PR, and then commit them in separate PRs.

Reply:
> I couldn't repro the issue personally, but a user tested my suggestion in [#196 (comment)](https://github.com/lstein/stable-diffusion/issues/196#issuecomment-1230846125) and reports that it worked, so it should be ok. Do you want to just add that in while merging this PR? Alternatively I could open a new one with just that change.

I made those changes and got the PR updated for ya. I can confirm your suggestion worked, and it's so much cleaner than the way I was trying to do it.